miner/builder.go

```diff
@@ -16,14 +16,15 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/prysmaticlabs/prysm/v4/crypto/bls"
+	"github.com/prysmaticlabs/prysm/v4/crypto/keystore"
```

Comment:
geth also has keystore package https://github.com/NethermindEth/go-ethereum/blob/master/accounts/keystore/keystore.go
but maybe there is a reason to not use it?